x-pack/plugins/lens/public/drag_drop/drag_drop.tsx

```diff
@@ -299,6 +314,19 @@ const DragInner = memo(function DragDropInner({
                 }
               } else if (e.key === keys.ESCAPE) {
                 dragEnd();
```

Comment:
I notice that we are triggering the "Movement cancelled" message even if the drag wasn't started. Consider checking that the drag is happening before calling dragEnd.